### PR TITLE
[Fix #2647] Allow xstr interpolations

### LINKED ISSUE #2647: Lint/LiteralInInterpolation incorrectly flags interpolation of backticks
### Steps to Reproduce

```bash
$ echo 'puts "Backticks #{`date`}"' > foo.rb
$ rubocop foo.rb 
```

### Observed Behavior

```
foo.rb:1:19: W: Literal interpolation detected.
puts "Backticks #{`date`}"'
                  ^^^^^^
```

### Expected Behavior

Interpolation of dynamic string literals should be allowed.

### Environment

```bash
$ rubocop -V
0.36.0 (using Parser 2.3.0.1, running on ruby 2.2.4 x86_64-linux)
```

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 * Add new `Style/SpaceAroundKeyword` cop. ([@lumeet][])
 
+### Bug fixes
+
+* [#2647](https://github.com/bbatsov/rubocop/issues/2647): Allow `xstr` interpolations. ([@segiddins][])
```
Comment (bbatsov) [inline CHANGELOG.md]: Mention the cop of the name affected here.

Comment (segiddins) [inline CHANGELOG.md]: Done

**Comment by bbatsov** [comment]
You'll have to rebase.

**Comment by segiddins** [comment]
rebased

**Comment by bbatsov** [comment]
And now it conflicts with another PR of yours I just merged. Guess you can just rebase them one by one when I merge one of them. Sorry about the inconvenience.

**Comment by alexdowad** [comment]
@segiddins If you are submitting a large number of improvements, you may save yourself a lot of trouble by batching several of them in a single PR. That's what I have started doing (out of necessity).

**Comment by bbatsov** [comment]
For him yes - for me not so much. :-) I'd rather be rebasing the PRs myself than having to deal with huge PRs.

**Comment by bbatsov** [comment]
I've merged this myself.

**Pull request closed without merging**